src/renderer/store/battle.store.ts

```diff
@@ -38,6 +42,16 @@ function removeFromSpectators(participant: Player) {
     if (index !== -1) battleStore.spectators.splice(index, 1);
 }
 
+function addBot(ai: EngineAI | GameAI, teamId: number) {
+    battleStore.teams[teamId].push({
+        id: battleWithMetadataStore.participants.length,
+        name: ai.name,
+        aiOptions: ai.options || {},
```

Comment:
`ai.options` contains the description of options that can be passed, it's for lobby to be able to render dialog to configure AI. `aiOptions` you assign to is what options should be set to what values and passed to engine. So it's entirely different information. Pass empty dict as in the previous version of code in the other place.

With current code, generated startscript is ivalid and engine can't start.

Reply:
Already fixed, will push some changes in a few minutes.

Reply:
Should be good now